### ISSUE TITLE
DatabaseSeeder task broken

### ISSUE THREAD

**Issue by Ultimater** [phanbook/phanbook#246]
See error in screenshot below:  

![broken-task](https://user-images.githubusercontent.com/1922199/29427099-982d5aea-833d-11e7-9bec-22a9ce2f0959.png)

The backtrace in the screenshot mentions line 35:
https://github.com/phanbook/phanbook/blob/667d07f9082bb871b73ba6265c66c12d370c9c10/core/modules/cli/tasks/DatabaseSeederTask.php#L35

This is caused by the following four pointing nowhere:

    use Phanbook\Databases\UsersSeeder;
    use Phanbook\Databases\PostsSeeder;
    use Phanbook\Databases\TagsSeeder;
    use Phanbook\Databases\PostsTagsSeeder;

Almost exactly 2 years ago, you find the last instance of them in the repo:  
Commit: [01226ab](https://github.com/phanbook/phanbook/tree/5d4927797946f5e4b0f5aa8595f166a7974dc9e9/databases)

They were removed 3 days later:  
Commit: [c71a76a](https://github.com/phanbook/phanbook/commit/c71a76a83a9efe57e6dd5bfcecacb617f148822e)

Right now the task simply fails due to the error. We could reinstate the seeding files so the task has something to do. and/or revise the task. We could also aim at a different directory structure altogether even.  I like the idea of going in the Laravel direction and have a "database" directory to toss in our migrations. I think the `schema` approach makes it hard to track database changes. What do you guys think?
1) Remove `/schema` and reinstate `/databases`?  
2) Add `/databases` so we have both `/databases` and `/schema`?  
3) Or leave the directory structure alone and remove the task?

The schema right now is a complete mess. You can't even run the two SQL files back-to-back without encountering SQL errors related to the primary key already being used with multiple tables. Once it fails, you try to run it again, and the tables already exists because it's not a transaction. Anyone have any reasons to keep `/schema`  or any reasons against reinstating [`/databases`](https://github.com/phanbook/phanbook/tree/5d4927797946f5e4b0f5aa8595f166a7974dc9e9/databases)? I think consistency is a better choice. Also if we're going to head in the Laravel direction, I think we should call it `/database` rather than `/databases` in the plural.

## **TL;DR**  

I want to drop `/schema`, reinstate `/databases` then drop the "s", renaming it to `/database`.

**Comment by Ultimater** [comment]
Hmm, I appear to be wrong about the schema not being able to be run back-to-back without errors.  

Travis has successfully been able to run:  
```yml
- echo 'CREATE DATABASE phanbook CHARSET=utf8 COLLATE=utf8_unicode_ci' | mysql -u root
- cat schema/phanbook.sql | mysql -uroot phanbook
- cat schema/examples.sql | mysql -uroot phanbook
```
I tried running the two back-to-back on a fresh database and it worked fine for both the master and develop branches.  Perhaps leave `/scheme` alone for now.
